### ISSUE TITLE
GeoJSONLayer mask problem when zooming

### ISSUE THREAD

**Issue by clemishow** [uber/deck.gl#2454]
#### Actual Result
![screenshot 2018-11-21 at 12 10 45](https://user-images.githubusercontent.com/9325482/48837776-8e15e400-ed86-11e8-8f90-a7023ef1e591.png)

#### Expected Result
![screenshot 2018-11-21 at 12 10 53](https://user-images.githubusercontent.com/9325482/48837781-90783e00-ed86-11e8-976f-d6ee1fbb33d6.png)

#### Reproduce Steps 
GeoJSON isochrone: 
[isochrone.geojson.zip](https://github.com/uber/deck.gl/files/2603982/isochrone.geojson.zip)

GeoJSON Mask: 
[mask.geojson.zip](https://github.com/uber/deck.gl/files/2604009/mask.geojson.zip)

Using turf (5.1.6) to create a mask. 
Deck.gl: 6.3.0.
react-map-gl: 4.0.2.

#### Code Example
```javascript
const initialViewState = {
  longitude: 2.354975,
  latitude: 48.854434,
  zoom: 11,
  pitch: 0,
  bearing: 0
};

class MapLayers extends Component {
  polyMask = (isochrone) => {
    const bboxPoly = turf.bboxPolygon([-180, -85, 180, 85]);
    const mask = turf.difference(bboxPoly, isochrone);
    return mask;
  }

  render() {
    const layer = new GeoJsonLayer({
      id: 'geojson-layer',
      data: this.props.isochrone && this.polyMask(this.props.isochrone.geometry),
      getFillColor: [160, 160, 180, 200],
      stroked: false,
      extruded: false,
      wireframe: true,
      lineJointRounded: true,
    });

    return (
      <DeckGL
        initialViewState={initialViewState}
        controller={true}
        layers={layer}
        width={window.innerWidth}
        height={window.innerHeight}
      >
        <InteractiveMap 
          mapboxApiAccessToken={'X'} 
          mapStyle={'X'} 
        >
        </InteractiveMap>
      </DeckGL>
    );
  }
}
```

**Comment by Pessimistress** [comment]
@georgios-uber This is due to auto-offset mode falsely assuming all features are close to the viewport center. The outer box `[-180, -85, 180, 85]` is projected as:

![image](https://user-images.githubusercontent.com/2059298/49104094-fefb4700-f231-11e8-8eb4-cf7cb30a21d7.png)

A cheap way to mitigate this could be ignoring the `project_uPixelsPerUnit2` adjuster if offset is larger than some threshold.

**Comment by Pessimistress** [comment]
@clemishow Thanks for reporting this issue. Basically the new projection mode that we shipped with v6.2 has a bug when rendering features this large (geospatially) at zoom >= 12. While we work on a fix, here are a few work-around options to unblock you:

1. If high-precision is not required, you can add the prop `coordinateSystem: COORDINATE_SYSTEM.LNGLAT_DEPRECATED` to your GeoJsonLayer to use the legacy projection mode
2. Instead of `turf.bboxPolygon([-180, -85, 180, 85])`, subtract from a bounding box that just envelopes the Paris area. You can add a few rectangles tiles around it to fill the rest of the world if needed.
3. Based on the current zoom level, switch between the world-sized feature and a Paris-sized feature.

**Comment by clemishow** [comment]
@Pessimistress Hello, thank you, I change the bbox in Paris and it works

**Comment by Pessimistress** [comment]
A fix is published in 6.3.1
